packages/astro/package.json

```diff
@@ -224,7 +224,7 @@
     "unified": "^11.0.4"
   },
   "engines": {
-    "node": ">=18.14.1",
+    "node": ">=18.20.1",
     "npm": ">=6.14.0"
```

Comment:
I realize we forgot to mention this in our documentation, but maybe we should outline our policy for updating package managers versions as well. I'm actually not quite sure Astro really installs on npm 6.14.0, and last I heard, some deps don't install correctly on Yarn 1.

(For context, Sharp requires `>=9.6.5` for `npm`, `>=3.2.0` for Yarn and `>=7.1.0` for `pnpm`, probably due to bugs with installing native deps.)

Reply:
You're making a good point. If we look at the [Node.js document](https://nodejs.org/en/about/previous-releases#release-schedule), the npm minimum version shipped by the maintenance LTS is `v10.5.0`, so I think it's perfectly safe to bump the `npm` version..